deploy/resources/addondeploymentconfig.yaml

```
@@ -30,4 +30,6 @@ spec:
     value: opentelemetrycollectors.v1beta1.opentelemetry.io
   # Enables Opentelemetry Instrumentation
   - name: userWorkloadInstrumentation
-    value: instrumentations.v1alpha1.opentelemetry.io
\ No newline at end of file
+    value: instrumentations.v1alpha1.opentelemetry.io
+  - name: incidentDetection
+    value: monitoring.rhobs
```

Unix files should end with a newline
```suggestion
    value: monitoring.rhobs

```